jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -154,7 +181,9 @@ final case class JmsProducerSettings(connectionFactory: ConnectionFactory,
     copy(acknowledgeMode = Option(acknowledgeMode))
 }
 
-final case class Credentials(username: String, password: String)
+final case class Credentials(username: String, password: String) {
+  override def toString = s"Credentials($username,${"*" * password.length})"
```

Comment:
*-ing out passwords since they would be printed out otherwise in error messages I added